### ISSUE TITLE
new keyword is warned for list-all-equatable-fields

### ISSUE THREAD

**Issue by ngoc-quoc-huynh** [bancolombia/dart-code-linter#271]
Migrating to Dart 3.13 and removing the type name from the constructor in combination with the `list-all-equatable-fields` lints the `new` keyword as a field.

```dart
class Foo extends Equatable {
  const Foo();

  @override
  List<Object> get props => []; 
}

class Foo extends Equatable {
  // Remove type name from constructor
  const new();

  @override
  List<Object> get props => []; // Triggers Missing declared class fields: new
}
```

**Comment by ngoc-quoc-huynh** [comment]
Can't seem to reproduce this warning. Maybe it was a UI bug in my IDE. Closing for now.
